content_scripts/cip.js

```diff
@@ -298,6 +310,7 @@ var cip = {
     retrieveAndFillPassword: function ()
     {
         cip.fillPasswordOnly = true;
+			mcCombs.fillPasswordOnly = true;
```

Comment:
please use the same tab/spaces as the rest of the document (in this case, spaces)